src/partials/latestBlock/latestBlock.vue

```diff
@@ -79,6 +79,17 @@ export default {
     .then((blockHeight) => this
     .$store
     .dispatch('blocks/getBlockFromHeight', blockHeight))
+  },
```

Comment:
Probably `mounted` handler can be removed by using `immediate: true` in `watch`, it will reduce code duplication
https://vuejs.org/v2/api/#watch